File: Latihan2java/Program/Human.java
package Latihan.Latihan2java.Program;

public class Human {
    private String nama;
    private String nik;
    private char gender;

    public Human(){
        this.nama = "";
        this.nik = "";
        this.gender = ' ';
    }

    public Human(String nama, String nik, char gender){
        this.nama = nama;
        this.nik = nik;
        this.gender = gender;
    }

    // Get-set nama
    public String getNama(){
        return this.nama;
    }

    public void setNama(String nama){
        this.nama = nama;
    }
    
    // Get-set nik
    public String getNIK(){
        return this.nik;
    }

    public void setNIK(String nik){
        this.nik = nik;
    }

    // Get-set Gender
    public char getGender(){
        return this.gender;
    }

    public void setGender(char gender){
        this.gender = gender;
    }
}

File: CPP/Program/Human.cpp
#include <iostream>
#include <string>

using namespace std;

class Human{

    private:

    string nama;
    string nik;
    char gender;

    public:
    
    // Constructor
    Human(){
    }

    Human(string nama, string nik, char gender){
        this->nama = nama;
        this->nik = nik;
        this->gender = gender;
    }

    // Get-set Nama
    void setNama(string nama){
        this->nama = nama;
    }

    string getNama(){
        return this->nama;
    }

    // Get-set NIK
    void setNIK(string nik){
        this->nik = nik;
    }

    string getNIK(){
        return this->nik;
    }

    // Get-set Gender
    void setGender(char gender){
        this->gender = gender;
    }

    char getGender(){
        return this->gender;
    }

    // Deconstructor
    ~Human(){
    }
};